client/examples/read_to_csv_all.py

```diff
@@ -0,0 +1,75 @@
+import sys
+sys.path.append('../src/ptprobe')
+
+import datetime
+import logging
+import threading
+import time
+import argparse
+import board
+from sinks import CsvSampleSink
+
+
+ports = ["COM1", "COM2"]
+boards = []
+sinks = []
+
+if __name__ == "__main__":
+    format = "%(asctime)s: %(message)s"
+    logging.basicConfig(format=format, level=logging.INFO, datefmt="%H:%M:%S")
+    
+    parser = argparse.ArgumentParser(description='Read PTProbe sensors over serial')
+    parser.add_argument('-m','--max-count', type=int, default=0,  
+            help='Maximum number of samples to record. Default 0 (no maximum)')
+    parser.add_argument('-t','--timeout', type=int, default=0,  
+            help='Collection time for sampling (s). Default is 0 (no timeout). The nominal sample rate is 5Hz.')
+    parser.add_argument('-p', '--port', default='/dev/ttyACM0',  
+            help='Serial port name. Default is /dev/ttyACM0.')
+    parser.add_argument('filename', help='CSV file for data output')
+    args = parser.parse_args()
+
+    logging.info("Starting demo")
+    logging.info(args)
+
+    logging.info("Creating sink {}".format(args.filename))
+    csv = CsvSampleSink(args.filename)
+    csv.open()
+    
+
+
+    for item in range(len(ports)):
```

Comment:
Yes, tried that originally but it wasn't iterating over the list properly so instead of fighting it at the time, just went through the numeric iteration - will refactor and test once the rig is back up.